domain/secretbackend/service/service.go

```diff
@@ -700,11 +700,6 @@ func (s *Service) RotateBackendToken(ctx context.Context, backendID string) erro
 	return errors.Trace(err)
 }
 
-// SetModelSecretBackend sets the secret backend for the given model UUID.
```

Comment:
The secret backend service should still do the actual work. We need to just pass the service to the model config facade. So the logic doesn't get moved to the model config service.